docs/index.md

```diff
@@ -126,7 +126,7 @@ If using SAM, you can include this SAR App as part of your shared Layers stack,
 
 === "Terraform"
 
-    ```ruby hl_lines="24-25"
+    ```terraform hl_lines="24-25"
```

Comment:
nice one, i looked initially at `hcl` but i didn't get the expected result. i should have thought about `terrafrom`